src/client/pages/LoginApp/LoginAppScreen.tsx

```diff
@@ -16,13 +17,32 @@ const PageWrapper = styled.div`
   height: 100vh;
   padding-left: 1rem;
   padding-right: 1rem;
-  color: ${colorsV3.gray500};
-  background-color: ${colorsV3.gray900};
+  color: ${colorsV3.gray700};
+  background-color: ${colorsV3.white};
+`
+
+const Header = styled.header`
+  display: flex;
+  justify-content: center;
+  padding-top: 1.25rem;
+  padding-bottom: 1.15rem;
+  background-color: ${colorsV3.white};
+
+  @media (min-width: 800px) {
```

Comment:
Maybe use [shared media queries](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/utils/mediaQueries.ts#L9) here:

```suggestion
import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
...
  ${MEDIUM_SCREEN_MEDIA_QUERY} {
```

Reply:
good catch.